transform/mattermost-analytics/models/marts/product/fct_active_users.sql

```diff
@@ -22,20 +22,28 @@ with metrics as (
 )
 select
     -- Client telemetry
-    m.*,
+    coalesce(m.daily_server_id, sas.daily_server_id) as daily_server_id
+    , coalesce(m.activity_date, sas.activity_date) as activity_date
+    , coalesce(m.server_id, sas.server_id) as server_id
+    {% for metric, column in column_map.items() %}
```

Comment:
q: do we need the for loop here as well since we had one in the `metrics` CTE above?